src/components/HeroBlock/HeroBlock.jsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+import HeroBlockComponent from "./HeroBlockComponent";
+import HeroBlockPicture from "./img/heroBlockPicture.png";
+
+function HeroBlock() {
+  return (
+    <HeroBlockComponent
+      title="Izbaudi profesijas ēnošanas piedzīvojumu un atklāj savu karjeras
+  potenciālu!"
+      subtitle_span="TryWork"
+      subtitle="ir platforma, kurā motivētas personas satiekas ar atsaucīgiem
+  Ēnu devējiem."
+      button_left_text="Reģistrējies kā Ēnotājs"
+      button_left_link="/registration/register-shadower"
+      button_right_text="Reģistrējies kā Ēnu devējs"
```

Comment:
Also you have a little bit too much props.
Maybe it worth to combine them by meaning, example would be: - 

1. **title + subtitle_span + subtitle**
2. **button_left_text + button_left_link**
3. **button_right_text + button_right_link**
4. **image_src + image_alt**

Reply:
How?

Reply:
It could be done like so:

```
const titleData = {
  title,
  subtitle,
  span
}

<HeroBlockComponent titleData={titleData} same with buttonLeft, buttonRight, imageData />
```